README.md

```diff
@@ -129,6 +129,9 @@ which are created during tenant initialization, the values of which can be custo
 and `DI_RAW_RECORDS_CHUNK_PARSED_PARTITIONS` env variables respectively.
 Default value - `1`.
 
+## Environment variables
+`MAX_NUM_EVENTS`:`100` - set max num events to consume
```

Comment:
For future reference, we should name the env var better so that it communicates its domain at batch journaling rather than the entire application.

Reply:
thank you! will update in later PR